integrations/appsflyer/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest>
```

Comment:
Can we add [this](https://github.com/rudderlabs/rudder-sdk-kotlin/blob/6908520d409c4a9d3b227a693c0010406eb1690e/integrations/adjust/src/main/AndroidManifest.xml#L2)? 
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```

Reply:
Let's try to ensure we have the same thing as added in other integrations module.